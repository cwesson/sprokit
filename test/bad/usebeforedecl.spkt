func test() : int32 {
	a = 1 + 2;
	var a : int32 = 12;
	if true {
		var b : int32 = 14;
	}
	a = b + 6;
	return 0;
}
