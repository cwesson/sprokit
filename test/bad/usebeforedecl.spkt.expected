Error on line 2:
  Variable a not declared.
Error on line 7:
  Variable b not declared.
Error on line 7:
  Cannot add unknown type and int8
Error on line 7:
  Cannot assign unknown type to int32
